packages/ruleset/src/functions/pagination-style.js

```diff
@@ -197,7 +197,7 @@ function paginationStyle(pathItem, path) {
     const pageTokenParam = params[pageTokenParamIndex];
     if (pageTokenParam.name !== 'start') {
       results.push({
-        message: `The "${pageTokenParam.name}" parameter should be named "start"`,
+        message: `The ${pageTokenParam.name} parameter should be named start`,
```

Comment:
I think that regardless of what we decide for wrapping names in quotes, we could wrap "start" in quotes here since we're actively prescribing a string for the user to use.